website_sale_hide_price/views/website_sale_template.xml

```diff
@@ -29,22 +29,29 @@
         </xpath>
         <xpath expr="//a[@id='add_to_cart']" position="attributes">
             <attribute name="t-if">
-                    website.website_show_price and not product.website_hide_price
-                </attribute>
+                website.website_show_price and not product.website_hide_price
+            </attribute>
         </xpath>
     </template>
     <template id="product_quantity" inherit_id="website_sale.product_quantity">
         <xpath expr="//div[hasclass('css_quantity')]" position="attributes">
             <attribute name="t-if">
-                    website.website_show_price and not product.website_hide_price
-                </attribute>
+                website.website_show_price and not product.website_hide_price
+            </attribute>
+        </xpath>
+    </template>
+    <template id="products_add_to_cart" inherit_id="website_sale.products_add_to_cart">
+        <xpath expr="//a[hasclass('a-submit')]" position="attributes">
+            <attribute name="t-if">
+                website.website_show_price and not product.website_hide_price
```

Comment:
This isn't working

Reply:
Check again with the new change to another template to see if it works because in my local it works fine, if not we will have to check it in another way.